content/millennium/r4/proprietary-codes-and-systems.md

```diff
@@ -21,8 +21,9 @@ The following are true for all Millennium proprietary codes:
 * The `code` value is the numeric code value as a string
 * The `display` value is the code value display
 * The `userSelected` value is set to true
+* When adding or updating data, if both proprietary and standard codes are populated for the specific request body field, the request will fail
 
-Example CodeableConcept displaying both standard and proprietary codes:
+Example read from CodeableConcept displaying both standard and proprietary codes:
```

Comment:
```suggestion
Example outbound CodeableConcept displaying both standard and proprietary codes:
```
Would this work better here?

Reply:
[updated](https://github.com/cerner/fhir.cerner.com/pull/313/files#diff-6fdcb00b5b1b6edd51b687794bea6b95R26)

Reply:
https://github.com/cerner/fhir.cerner.com/pull/313/commits/581029c3907bdc5324a38ae544ae9f9f853942d7